audformat/core/utils.py

```diff
@@ -343,17 +343,10 @@ def to_segmented_index(
     else:
         index = obj
 
-    df = index.to_frame(index=False)
-    df[define.IndexField.START] = pd.to_timedelta(0)
-    df[define.IndexField.END] = pd.NaT
-
-    index = pd.MultiIndex.from_frame(
-        df,
-        names=[
-            define.IndexField.FILE,
-            define.IndexField.START,
-            define.IndexField.END,
-        ]
+    index = segmented_index(
+        files=list(index),
+        starts=[0] * len(index),
+        ends=[pd.NaT] * len(index),
```

Comment:
I think it is in general better to use our own `segmented_index` function here so we don't have to update this function if we update `segmented_index()`. Maybe it is not as efficient as the previous solution, but hopefully this will not a big deal.

Reply:
Agree